### DISCUSSION TITLE
Docker image for Java

### DISCUSSION THREAD

**Discussion by liuyanqing119** [OpenDDS/OpenDDS#4285] [General]
I need a docker image for Java, can the docker image in [quick start ](https://opendds.org/quickstart/GettingStartedDocker.html)be used in Java programs?
Do I need to compile it myself via dockerfile?

**Comment by iguessthislldo** [comment]
Java isn't enabled by default and it's not in the Dockerfile so yea you'd have to edit the Dockerfile and build it yourself. I think all you'd have to do is add whatever JDK package for that Ubuntu you want to use to the `apt-get` line and add `--java` to `configure`.

**Comment by jrw972** [comment]
Please submit a PR with your proposed changes to the Dockerfile.  Perhaps we can either 1) parameterize the Dockerfile to enable Java or 2) create a new Dockerfile for Java.  In either case, this sounds like a good addition to incorporate into the existing Docker image build (https://github.com/OpenDDS/OpenDDS/actions/workflows/docker_image.yml).
